agent_secret_manager/src/subject.rs

```diff
@@ -53,55 +141,42 @@ impl Verify for Subject {
                     }
                     _ => None,
                 })
-                .ok_or(anyhow::anyhow!("Failed to decode public key for DID URL: {}", did_url))
+                .ok_or(anyhow!("Failed to decode public key for DID URL: {did_url}"))
```

Comment:
This should be wrapped in backticks for consistency with other log messages above.